addons/libs/images.lua

```diff
@@ -10,6 +10,8 @@ local meta = {}
 
 saved_images = {}
 local dragged
+local clicked
+local last_pos = {}
```

Comment:
Declared this as a table right away, unlike dragged, so that line 415 can use the new function to assign the values without erroring out due to the function trying to declare keys on a var that was not yet declared as a table.

Reply:
Check `if clicked then` later on, this is a bug and only works because Lua is weakly typed.